templates/focus-ios.xliff

```diff
@@ -729,6 +749,74 @@
         <source>URL to open</source>
         <note>Label for input to set a favorite URL to be opened by Siri.</note>
       </trans-unit>
+      <trans-unit id="Tip.Biometric.Title">
+        <source>Lock %@ when a site is open:</source>
+        <note>Text for a label that indicates the title for biometric tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.BiometricFaceId.Description">
+        <source>Turn on Face ID</source>
+        <note>Text for a label that indicates the description for biometric Face ID tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.BiometricTouchId.Description">
+        <source>Turn on Touch ID</source>
+        <note>Text for a label that indicates the description for biometric Touch ID tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Release.Description">
+        <source>Read more about this and other updates to %@.</source>
+        <note>Text for a label that indicates the description for release tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Release.Title">
+        <source>Why yes, we do have a fresh new look!</source>
+        <note>Text for a label that indicates the title for release tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.RequestDesktop.Description">
+        <source>Page Actions &gt; Request Desktop Site</source>
+        <note>Text for a label that indicates the description for request desktop tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.RequestDesktop.Title">
+        <source>Want to see the full desktop version of a site?</source>
+        <note>Text for a label that indicates the title for request desktop tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.ShareTrackers.Description">
+        <source>%@ trackers blocked so far</source>
+        <note>Text for a label that indicates the description for share trackers tip.</note>
```

Comment:
* same comment about placeholders
* plurals not properly supported, let's update it to this workaround: "Trackers blocked so far: %@"
* Also, nit in comment: "the title *to* share trackers tip"

Reply:
@Delphine  We only show this if there are multiple trackers blocked. It will never be 1 tracker always multiple. Will that work?

Reply:
Unfortunately languages have many different ways to support plurals depending on what number we are calling out.
I don't know how to fix this on a technical level. So this has been the workaround we've always used for iOS products

Reply:
I talked to flod and I understand this issue much better now. I've added the following comment:

> For locales where we would need plural support, please feel free to translate this string as `Trackers blocked so far: %@`.